### ISSUE TITLE
"DEMO" slides need unfolding, or, ideally, videos

### ISSUE THREAD

**Issue by cascremers** [tamarin-prover/teaching#1]
There are some "DEMO" slides in the deck which are hard to explain in speaker notes, but typically there are none. These should be sensibly expanded. Videos (with subtitles) would be ideal.

**Comment by rsasse** [comment]
Agreed, there are speaker notes but they are very minimal and more reminders to someone knowing the example rather than for a first-time presenter. Videos would of course be nice!

**Comment by rsasse** [comment]
Additionally we should mention on each slide titled "DEMO" which file we are supposed to look at now.

**Comment by huahuaLover** [comment]
So, is there a video now？

**Comment by jdreier** [comment]
Not for the examples from the slides, but David Wong created a video for a different example: https://cryptologie.net/article/404/tamarin-prover-introduction/
